packages/api/src/routes/medical/schemas/patient.ts

```diff
@@ -38,3 +46,24 @@ export function schemaUpdateToPatientData(input: PatientUpdate): PatientData {
 export function schemaDemographicsToPatientData(input: Demographics): PatientData {
   return schemaCreateToPatientData(input);
 }
+
+const coverageAssessmentPatientSchema = z.object({
```

Comment:
Couldn't we use the existing schema for patient create?

Reply:
I thought about this, and it's a question of how we plan on using the endpoint. If the idea is to map CSVs into the endpoint, the argument is that we should keep the payload flat (more closely aligns with a CSV). If the argument is that we're *essentially* starting with patient create/update so the payload should match these commands and converting form CSV to nested address / contact structure is the responsibility of the caller) then I agree.

Reply:
I see your point. So it seems we're sending the payload based on the CSV, right?

Reply:
Yes.